packages/profile-sync-controller/src/controllers/user-storage/accounts/user-storage.ts

```diff
@@ -0,0 +1,55 @@
+import type { InternalAccount } from '@metamask/keyring-api';
+
+import {
+  USER_STORAGE_VERSION_KEY,
+  USER_STORAGE_VERSION,
+  LOCALIZED_DEFAULT_ACCOUNT_NAMES,
+} from './constants';
+
+export type UserStorageAccount = {
+  /**
+   * The Version 'v' of the User Storage.
+   * NOTE - will allow us to support upgrade/downgrades in the future
+   */
+  [USER_STORAGE_VERSION_KEY]: typeof USER_STORAGE_VERSION;
```

Comment:
Future - hmmm I think we may need to have different version for each different feature.

We can couple this work as part of the versioning support ticket.